app/routes/attribute.routes.js

```diff
@@ -5,30 +5,22 @@
  */
 var attributeController = require('../controllers/attribute.controller');
 var config = require('../config/config');
+var responseHandler = require('../utils/response.handler');
 
 module.exports = function(routes) {
 
     // create an attribute
     routes.post('/', function(request, response) {
         attributeController.createNewAttribute(request.body)
-            .then(function(attribute) {
-                return response.json(attribute);
-            })
-            .catch(function(error) {
-                return response.status(error.status || 500).send(error.message);
-            });
+            .then(response.json.bind(response))
```

Comment:
I agree that the methods become short and sweet like this, but I wonder if it wouldn't be more readable to create a small private helper function called for example `sendJsonResponse` and pass that to `then`. That way it'd read something like "create new attribute, then send JSON response". What do you think?